src/Microsoft.Health.Dicom.SqlServer/Features/ChangeFeed/SqlChangeFeedStoreV1.cs

```diff
@@ -0,0 +1,119 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Data.SqlClient;
+using Microsoft.Health.Dicom.Core.Features.ChangeFeed;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Client;
+using Microsoft.Health.SqlServer.Features.Storage;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.ChangeFeed
+{
+    public class SqlChangeFeedStoreV1 : ISqlChangeFeedStore
```

Comment:
The supported Schema is V3-v4 before. Why do we need to implement all the historical versions?

Can we reuse classes for versions that are not changing anything?

This smells to me

Reply:
Is that possible that someone will be in V1 or V2. Should we migrate them?

Reply:
We were trying to follow the existing pattern but agree these files are not necessary. We will address this in a future refactor.